misc/org.projecthamster.hamster-gtk.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
yes, but I do not like this pattern too much. The way I see it ``data`` should be for application specific data that is actually used by the app. What the current ``misc`` dir contains however is just helpers for the deployment environment that have nothing to do with the app in a stricter sense. These are just things that should make integrating the app into a wider environment easier. I really don't mind a different name than ``misc`` though, its not that important to me at this stage - just ``data`` is probably not really my cup of tea.